pkg/asset/installconfig/vsphere/validation.go

```diff
@@ -411,3 +429,56 @@ func validateVcenterPrivileges(validationCtx *validationContext, fldPath *field.
 	}
 	return field.ErrorList{}
 }
+
+func ensureLoadBalancer(installConfig *types.InstallConfig, fldPath *field.Path) field.ErrorList {
+	dialTimeout := time.Second
+	tcpTimeout := time.Second * 10
+	tcpContext, cancel := context.WithTimeout(context.TODO(), tcpTimeout)
+	defer cancel()
+
+	errList := field.ErrorList{}
+	var lastErr error
+	errorCount := 0
+
+	apiURI := fmt.Sprintf("api.%s.%s", installConfig.ObjectMeta.Name, installConfig.BaseDomain)
```

Comment:
This can be simplified some using `installConfig.ClusterDomain()` https://github.com/openshift/installer/blob/master/pkg/types/installconfig.go#L195-L198